app/src/main/java/io/homeassistant/companion/android/settings/SettingsFragment.kt

```diff
@@ -429,6 +434,7 @@ class SettingsFragment constructor(
         val success = result == Authenticator.SUCCESS
         val switchLock = findPreference<SwitchPreference>("app_lock")
         switchLock?.isChecked = success
+        findPreference<SwitchPreference>("app_lock_home_bypass")?.isVisible = success
```

Comment:
We should probably hide or keep this disabled if the user did not configure home wifi SSIDs yet. That way the behavior follows the rest of the settings page like internal connection.

Reply:
Thanks for the feedback. I've made changes to keep this setting disabled when no home networks are configured.